apps/dc_tools/odc/apps/dc_tools/stac_api_to_dc.py

```diff
@@ -216,10 +240,15 @@ def cli(
     if limit is not None:
         config["max_items"] = limit
 
+    if rewrite_assets is not None:
+        rewrite = list(rewrite_assets.split(";"))
```

Comment:
in your help `,` character is used, but this one is `;`, I'd say `,` is more command line friendly

Reply:
Ok, fixing.